Models/User/UserDAO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace project_backend.Models.User
+{
+    public class UserDAO
```

Comment:
Ar fi de preferat sa punem ca atribut [Table()] ca sa nu avem ca nume in DB o tabela cu numele "UserDAO"
```suggestion
    [Table("users")]
    public class UserDAO
```